src/js/tabs/osd.js

```diff
@@ -1332,6 +1332,11 @@ OSD.constants = {
             text: 'osdWarningTextRssiDbm',
             desc: 'osdWarningRssiDbm'
         },
+        OVER_CAP: {
+            name: 'OVER_CAP',
+            text: 'osdWarningTextOverCap',
+            desc: 'osdWarningOverCap'
```

Comment:
As Sonar issue, add a trailing comma to the latest element.

Reply:
Will do, thought it'd be inconsistent with the other statements above it that don't have the trailing comma

Reply:
Yes, we are trying to unify the code, so the new code must be according to our Sonar rules. Maybe in a future PR we fix the rest of Sonar issues of this file ;)